contrib/kind.sh

```diff
@@ -834,22 +845,28 @@ build_ovn_image() {
     # store in local registry
     if [ "$KIND_LOCAL_REGISTRY" == true ];then
       echo "Pushing built image to local $OCI_BIN registry"
-      $OCI_BIN push "${OVN_IMAGE}"
+      $OCI_BIN push "${push_args}" "${OVN_IMAGE}"
     fi
   # We should push to local registry if image is not remote
   elif [ "${OVN_IMAGE}" != "" -a "${KIND_LOCAL_REGISTRY}" == true ] && (echo "$OVN_IMAGE" | grep / -vq); then
     local local_registry_ovn_image="localhost:5000/${OVN_IMAGE}"
     $OCI_BIN tag "$OVN_IMAGE" $local_registry_ovn_image
     OVN_IMAGE=$local_registry_ovn_image
-    $OCI_BIN push $OVN_IMAGE
+    $OCI_BIN push "${push_args}" "$OVN_IMAGE"
```

Comment:
@ormergi one last ask I am not bash expert so feel free to not do this shouldn't it be "${OVN_IMAGE}" here? the braces don't matter?

Reply:
Beaches usually doesnt matter unless some string manipulation or slicing is necessary.
The convention in the file is in consistent and there is no bash linter to enforce it, I am not sure whats better :sweat_smile:

Reply:
I removed the braches in the changes of this PR, keep this PR changes consistent as least.

Reply:
yeah this file is not having bash linter :( but yea I don't think in the end like you said it matters with string manipulation for bash for braces and no braces are valid looks like
@coderabbitai what do you think?